.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve. Report bugs found while using the project
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Description
+<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Actual Behavior
+<!--- Tell us what happens instead -->
+
+## Possible Fix
+<!--- Not obligatory, but suggest a fix or reason for the bug -->
+
+## Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+1.
```

Comment:
@kvedala Is there any need for the numbering 1,2,3,4?

Reply:
doesn't hurt .. it is the same protocol being used in the C-Plus-Plus repo. Hence, helps to maintain consistency

Reply:
also, helps the reviewers to refer to point# in the comments if the issue authors do not follow the norms and standards